src/Compilers/CSharp/Portable/Binder/ExecutableCodeBinder.cs

```diff
@@ -19,12 +20,14 @@ internal sealed class ExecutableCodeBinder : Binder
     {
         private readonly Symbol _memberSymbol;
         private readonly CSharpSyntaxNode _root;
+        private readonly Func<Binder, CSharpSyntaxNode, Binder> _rootBinderAdjusterOpt;
```

Comment:
>private [](start = 8, length = 7)

It would be very helpful to have some documentation around this, either here or on the constructor.